server/tilesource/tiff_reader.py

```diff
@@ -248,6 +254,22 @@ def _loadMetadata(self):
         self._imageWidth = info.get('imagewidth')
         self._imageHeight = info.get('imagelength')
         self.parse_image_description(info.get('imagedescription', ''))
+        units = {2: 25.4, 3: 10}
```

Comment:
Where do these numbers come from?

Reply:
I'll add a comment (it is from the Tiff specification).